app/controllers/articles.controllers.js

```diff
@@ -14,17 +14,24 @@ exports.getArticles = (req, res, next) => {
 
 exports.getArticles = (req, res, next) => {
     const { topic } = req.query
-
-    Promise.all([
+    
+    Promise.allSettled([
```

Comment:
Oh! I've never seen someone use an allSettled before - well done on this usecase